ansible/roles/common/templates/conf/output/01-es.conf.j2

```diff
@@ -1,3 +1,19 @@
+{% if enable_caso | bool and inventory_hostname in groups['caso'] %}
+<match apel.events>
+    @type copy
+    <store>
+       @type elasticsearch
+       host {% raw %}{{ elasticsearch_address }}
+{% endraw %}
+       port {% raw %}{{ elasticsearch_port }}
+{% endraw %}
```

Comment:
Are you sure these should be raw?

Reply:
Hmm, good point. This does look wrong, I just copied this across from the original patch.

Reply:
The bizaare thing is that this works as is:
```
<match apel.events>
    @type copy
    <store>
       @type elasticsearch
       host 192.168.33.2
       port 9200
       logstash_format true
       logstash_prefix apel
       flush_interval 15s
       reload_connections false
       reconnect_on_error true
    </store>
</match>
```

Reply:
This is without:
```
# Included from conf/input/99-caso.conf.j2:
<source>
  @type tcp
  tag apel.events
  port 24224
  bind 127.0.0.1
  format /^(?<Payload>.*)$/
  emit_unmatched_lines true
</source>
```